ntoskrnl/mm/ARM3/pfnlist.c

```diff
@@ -1227,6 +1231,13 @@ MiDecrementReferenceCount(IN PMMPFN Pfn1,
     /* PFN lock must be held */
     MI_ASSERT_PFN_LOCK_HELD();
 
+    /* Hack for RosMm */
```

Comment:
Can you maybe quickly explain what this does in the comment? (the "AWE allocation")